build.gradle

```diff
@@ -42,14 +42,25 @@ version '1.0.0-SNAPSHOT'
 
 publishing {
     publications {
-        group = "org.opensearch"
+        group = "org.opensearch.sdk"
```

Comment:
Would it make sense to specify at one place and pick it from there throughout the gradle file? The reason being it can be easily missed. Better to have single place of truth?
Your call.

Reply:
I like that sure!!